src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -115,11 +115,55 @@ static View()
             RegisterAccessibilityDelegate();
         }
 
+        /// <summary>
+        /// Accessibility mode for controlling View's Accessible implementation.
+        /// It is only relevant when deriving custom controls from View directly,
+        /// as classes derived from CustomView (or any of its subclasses) get the
+        /// Custom mode by default.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum ViewAccessibilityMode
```

Comment:
How about moving this enum declaration out of class?
In my opinion, `View.ViewAccessibilityMode` feels like a duplicate a lot, for example,
```
/// Current
var view = new View(View.ViewAccessibilityMode.Custom)
```
If you move it out of class, it can be simple.
```
/// Suggestion1
var view = new View(ViewAccessibilityMode.Custom)
```

Furthermore, if you rename it to `AccessibilityMode` the code will be much simpler.
```
/// Suggestion2
var view = new View(AccessibilityMode.Custom)
```

Could you consider these?
I understand this patch is just for back-porting, so I'd like you to create a new patch if you want to modify it.

Reply:
Thank you for the explaination.
As far as I understand, it is for `FluxView`. If that's the only case, I think it's okay to leave it as it is now.